### ISSUE TITLE
Update video_player to 2.0.0

### ISSUE THREAD

**Issue by nehaalrejam** [sbis04/video_trimmer#92]
Thanks a lot for the awesome pluginn!

Can we please upgrade the video_player plugin dependency to the latest 2.0.0 since on upgrading flutter sdk to the latest stable branch I get this

```
Because video_trimmer 0.3.5 depends on video_player ^1.0.1 and no versions of video_trimmer match >0.3.5 <0.4.0, video_trimmer ^0.3.5 requires video_player ^1.0.1.
```

Thanks!

**Comment by sbis04** [comment]
Currently, we haven't yet migrated this package to null-safety as some dependencies that this package is using haven't released their null-safety versions (Ref: #90 ).

You may use the lower version of the `video_player` package if possible, or you may try overriding the `video_player ` dependency, something like this:

```yaml
dependencies:
   # ...

# Add the following to your pubspec.yaml
dependency_overrides:
   video_player: ^2.0.0
```

**Comment by lvlrSajjad** [comment]
https://github.com/sbis04/video_trimmer/pull/98
